features/202-session-management/api.md

```diff
@@ -0,0 +1,119 @@
+# Session Management APIs
+
+## SDK API
+```typescript
+// Field value would be empty string if not available
+interface UserAgent {
+    raw: string;
+    name: string; // browser name or app ID
+    version: string; // browser version or app version
+    os: string;
+    osVersion: string;
+    device: string;
+}
+
+// Field value would be empty string if not available
+interface Device {
+    name: string; // device name
+}
+
+interface Session {
+    id: string;
+    identityID: string;
+    createdAt: Date;
+    lastAccessedAt: Date;
+    createdByIP: string;
+    lastAccessedIP: string;
+    userAgent: UserAgent;
+    device: Device?;
```

Comment:
device is part of UserAgent?

Reply:
I'll rename it to `deviceModel` for clarity.